### PR TITLE
DM-47303: Add fgcmcal configurations for LSSTComCam

### PR DESCRIPTION
**Pull request by erykoff**
This supports ugriz so far.

### CODE REVIEW

File: config/comCam/fgcmFitCycle.py [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+    "g_01": "g",
+    "r_03": "r",
+    "i_06": "i",
+    "z_03": "z",
+    "y_04": "y",
+}
+
+config.outfileBase = "fgcmLSSTComCam"
+config.bands = ["u", "g", "r", "i", "z"]
+config.fitBands = ["u", "g", "r", "i", "z"]
```
Comment (sr525) [inline config/comCam/fgcmFitCycle.py:14]: Do these need a y?

File: config/comCam/fgcmFitCycle.py [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+    "u": 0.15,
+    "g": 0.05,
+    "r": 0.05,
+    "i": 0.05,
+    "z": 0.05,
+    "y": 0.05,
+}
+config.autoPhotometricCutNSig = 3.0
+config.autoHighCutNSig = 3.0
+# Fit aperture corrections with only 2 bins to exercise the code.
```
Comment (sr525) [inline config/comCam/fgcmFitCycle.py:56]: Is this comment relevant still?

File: config/comCam/fgcmFitCycle.py [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+
+config.sedterms = fgcmcal.SedtermDict()
+config.sedterms.data = {
+    "u": fgcmcal.Sedterm(primaryTerm="ug", secondaryTerm="gr", constant=0.5,
+                         extrapolated=True, primaryBand="u", secondaryBand="g", tertiaryBand="r"),
+    "g": fgcmcal.Sedterm(primaryTerm="gr", secondaryTerm="ri", constant=1.5),
+    "r": fgcmcal.Sedterm(primaryTerm="gr", secondaryTerm="ri", constant=0.9),
+    "i": fgcmcal.Sedterm(primaryTerm="ri", secondaryTerm="iz", constant=1.1),
+    "z": fgcmcal.Sedterm(primaryTerm="iz", secondaryTerm="ri", constant=1.0,
+                         extrapolated=True, primaryBand="z", secondaryBand="i", tertiaryBand="r")
```
Comment (sr525) [inline config/comCam/fgcmFitCycle.py:85]: Are primary and secondary bands the correct way round?

File: config/comCam/fgcmFitCycle.py [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+config.useRepeatabilityForExpGrayCutsDict = {
+    "u": False,
+    "g": False,
+    "r": False,
+    "i": False,
+    "z": False,
+    "y": False,
+}
+config.sigFgcmMaxEGrayDict = {
+    "u": 0.15,
```
Comment (sr525) [inline config/comCam/fgcmFitCycle.py:148]: Is this one supposed to be different to the others?

File: config/comCam/fgcmFitCycle.py (hunk 1 above)
Comment (erykoff) [reply config/comCam/fgcmFitCycle.py:14]: We don't support y band yet.  All the stuff below is a placeholder.

File: config/comCam/fgcmFitCycle.py (hunk 3 above)
Comment (erykoff) [reply config/comCam/fgcmFitCycle.py:84]: Yes, they are.  This is start by using the z band and adjust from there.  z is primary to the z.

File: config/comCam/fgcmFitCycle.py [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+    "g": fgcmcal.Sedterm(primaryTerm="gr", secondaryTerm="ri", constant=1.5),
+    "r": fgcmcal.Sedterm(primaryTerm="gr", secondaryTerm="ri", constant=0.9),
+    "i": fgcmcal.Sedterm(primaryTerm="ri", secondaryTerm="iz", constant=1.1),
+    "z": fgcmcal.Sedterm(primaryTerm="iz", secondaryTerm="ri", constant=1.0,
+                         extrapolated=True, primaryBand="z", secondaryBand="i", tertiaryBand="r")
+    # "y": fgcmcal.Sedterm(primaryTerm="zy", secondaryTerm="iz", constant=1.0,
+    #                      extrapolated=True, primaryBand="y",
+    #                      secondaryBand="z", tertiaryBand="i")
+}
+
```
[the remark this answers is not in the document]
Comment (erykoff) [reply config/comCam/fgcmFitCycle.py:91]: We can uncomment when it supports y band.  I added a comment.

File: config/comCam/fgcmFitCycle.py (hunk 4 above)
Comment (erykoff) [reply config/comCam/fgcmFitCycle.py:151]: Yes!  Comment added.

**Pull request merged**